addons-web-sdk/README.md

```diff
@@ -19,7 +19,10 @@ npm install @googleworkspace/meet-addons
 ```
 
 For TypeScript users, TypeScript definitions are packaged with the module. There
-is no need to install any modules from DefinitelyTyped.
+is no need to install any modules from DefinitelyTyped. Also ensure that you use
+a ModuleResolution strategy in your `tsconfig.json` that [supports package.json 
+`exports`](https://www.typescriptlang.org/tsconfig/#moduleResolution) (likely
+you want to use one of 'node16', 'nodenext' or 'bundler').
```

Comment:
```suggestion
`exports`](https://www.typescriptlang.org/tsconfig/#moduleResolution).
```

Inclined to not include the node16, nodenext, etc. because it will quickly get outdated and will need frequent updates.

Reply:
+1